src/OpenLoco/Scenario.cpp

```diff
@@ -142,9 +149,25 @@ namespace OpenLoco::Scenario
     // 0x0049685C
     void initialiseDate(uint16_t year)
     {
-        registers regs;
-        regs.ax = year;
-        call(0x0049685C, regs);
+        // NB: this base value was already 1800 in Locomotion.
+        uint32_t baseYear = year - 1800;
+        uint32_t dayCount = (baseYear / 4) * (365 * 3 + 366);
```

Comment:
Wouldn't something like this be a *lot* clearer? (also, it's an initialisation function. I doubt the performance cost of the loop matters)
```
for(int y=1800; year<1800; year++) {
    days +=365;
    if(isLeapYear(year) {
        days += 1;
    }
}
```

Reply:
Yeah except y < year

Reply:
It certainly looks a bit easier to read, yes, but I'm not 100% convinced. This is roughly what you had in mind, right? https://godbolt.org/z/oqajne

Reply:
Yeah I really don't care about performance for an initialiser function. Readability is more important to me.

Reply:
I've pushed a commit with the changes.